src/cascade/executor/epiviz_runner.py

```diff
@@ -559,8 +574,8 @@ def one_location_set(ec, settings, posterior_draws_of_previous_fit):
     ec.dismodfile = write_dismod_file(mc, ec, ec.parameters.db_file_path)
     sampled_fit = None
     if not ec.parameters.db_only:
-        run_dismod(ec.dismodfile, "init")
-        run_dismod_fit(ec.dismodfile, has_random_effects(mc))
```

Comment:
Again, opposite direction.

Reply:
But maybe you want the context to take care of whether it's a file we run? Unsure.

Reply:
I was starting to juggle a lot of mysterious parameters for these functions so it made sense to me to pack some of them into the context object. But if we want to use the contexts less then that is the wrong motion. I don't feel strongly either way.

Reply:
This seems good here. I want to move to a point where I can guess which information is where. That needs to be addressed elsewhere, not this PR.